skaffold.yaml

```diff
@@ -130,27 +119,24 @@ profiles:
       - example/provider-local/seed-kind2/skaffold
 
 ---
-apiVersion: skaffold/v2beta25
+apiVersion: skaffold/v2beta29
 kind: Config
 metadata:
   name: gardenlet
 build:
   local:
-    useBuildkit: true
-    concurrency: 3
+    concurrency: 0
   artifacts:
   - image: eu.gcr.io/gardener-project/gardener/gardenlet
-    docker:
-      dockerfile: Dockerfile.local
-      target: gardenlet
+    ko:
+      main: ./cmd/gardenlet
+      fromImage: eu.gcr.io/gardener-project/alpine-openvpn:3.16.2
```

Comment:
Where is this image coming from? 👀

Reply:
I've pushed it manually. Do you think we need to build it in CI?

Reply:
I think that it will no longer be necessary after we merge gardener/gardener#6556

Reply:
Cool, interesting. Should we merge #6556 first then?

Reply:
Hmm, we still need to build the base image first before `ko` can start building the image.
I think, if we don't want such a static `fromImage` reference, the best option is to drop the dependency towards the `openvpn` binary.

Reply:
I get it now. Thanks for explaining! Regarding the "openvpn-base" image I think that we should not build and push it to a image registry on every PR/main branch update. So I would vote to have it manually updated or setup in a way in which the image will not be built constantly.

Reply:
ref https://github.com/gardener/gardener/issues/6561

Reply:
> So I would vote to have it manually updated or setup in a way in which the image will not be built constantly.

Ok, then let's keep it like it is and update it manually.
I added a corresponding comment to `skaffold.yaml`.